src/components/details/ProductView.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from "react";
+
+import "./ProductView.scss";
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+
+import { addToCart } from "../redux_api/actions/cartActions";
+import { Link } from "react-router-dom";
+
+function ProductView({ product }) {
+  const [quantity, setQuantity] = useState(1);
+
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const { id } = product;
+
+  const addToItemCart = () => {
+    dispatch(addToCart(id, quantity));
+    navigate("/cart");
+  };
+
+  const buyNow = () => {
+    dispatch(addToCart(id, quantity));
+    navigate("/address");
+  };
+
+  return (
+    <div className="productinfo-main-container">
+      <div className="productinfo-left-container">
+        <img src={product.image} alt="nothing" />
+      </div>
+      <div className="productinfo-right-container">
+        <h3>{product.name}</h3>
+        <p className="productview-price">₿ {product.id}</p>
+        <table>
```

Comment:
Can we use patternfly table here? https://www.patternfly.org/components/table

Reply:
+1

Reply:
removed table and used grid layout from pattenfly for alignment